package.json

```diff
@@ -85,6 +84,7 @@
     "@types/node-fetch": "^2.3.7",
     "@types/openfin": "^43.0.1",
     "@types/webpack-dev-middleware": "^2.0.3",
+    "eslint": "^6.8.0",
```

Comment:
How come this is now a dev dependency? Is this not needed within each of the downstream projects for the linter to run?

Reply:
Oops left the `-D` in when upgrading.